pkg/workload/tpcc/generate.go

```diff
@@ -34,20 +35,26 @@ const (
 	maxOrderLinesPerOrder    = 15
 
 	originalString = "ORIGINAL"
-	wYtd           = 300000.00
-	ytd            = 30000.00
 	nextOrderID    = 3001
-	creditLimit    = 50000.00
-	balance        = -10.00
-	ytdPayment     = 10.00
 	paymentCount   = 1
 	deliveryCount  = 0
 )
 
+func makeDecimal(value, scale int32) apd.Decimal {
```

Comment:
nit: it's a bit confusing that this helper passes scale as a negative value while `types.Decimal` passes it as a positive value. Can't win them all i guess. Perhaps add a comment to clarify this?

Reply:
```
func makeDecimal(value, exponent int32) apd.Decimal {
  var result apd.Decimal
	result.SetFinite(int64(value*int32(math.Pow10(exponent))), -exponent)
	return result
}
```
This only allows defining integer decimals.

I guess I could do something like
```go
func makeDecimal(value float64, scale int32) apd.Decimal {
}
```